packages/react-components/src/components/Link.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import cx from 'classnames';
+
+export interface LinkProps
+  extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+  bold: boolean;
+}
+
+const baseClass = 'lc-link';
+
+/**
+ * Simple component which renders an `<a>` element.
+ *
+ * Place the `<Link>` inside a `<Text>` component or pass a custom className to set the typography styles.
+ *
+ * Use `<Button kind="link">` to act as a `<button>` - <a target="_self" href="/story/components-button--kinds">see the story</a>.
+ *
+ * Use `<Button href="">` to display as a `<Button>` - <a target="_self" href="/story/components-button--link">see the story</a>.
+ */
+export const Link: React.FC<LinkProps> = ({
+  bold = false,
```

Comment:
Was there any discussion inside the team about style-like props? For example `bold` is a prop that could be replaced with a proper `className`. Do we have a policy to avoid styling by props? And if we want to keep `bold` prop because it's "commonly used", then why not `fontSize` or `color` ?

Reply:
The question is - do you want to force user to remember "proper" className or provide the prop `bold` which in fact adds another className (hides implementation details). 

This is some "kind" of Link. On the other hand, I'm wondering if we shouldn't get rid of it at all and provide text-decorations from `<Text>` component.

I would love to know the opinion of other devs from Desgin System @marcinsawicki @MichalPaszowski

Reply:
IMHO, decorators from the Text component will fit most of the use cases and make sense.

Reply:
Agree with @MichalPaszowski